unittests/AST/ASTImporterTest.cpp

```diff
@@ -814,6 +814,60 @@ TEST(ImportDecl, ImportFunctionTemplateDecl) {
               has(returnStmt(has(integerLiteral(equals(2)))))))))))))))))));
 }
 
+TEST(ImportDecl, DISABLED_ImportTemplateDefaultArgument) {
```

Comment:
Nobody (even I) has noticed that this test case does not belong here. This is a test for #310 .

Reply:
Don't worry, this is TDD, first we have the test, then in an other PR we make it pass! :)